Discourages multiple outputs writing to one dataset (#63851)
Azure/azure-content: articles/stream-analytics/power-bi-output.md

Power BI uses the first-in, first-out (FIFO) retention policy. Data will collect in a table until it hits 200,000 rows.

> [!NOTE]

> > Using multiple outputs to write to the same dataset is generally discouraged as it can cause several issues. Each output can try to create the Power BI dataset independently which can result in multiple datasets with the same name being created. Additionally, if the outputs do not have consistent schemas, this will cause the dataset to change schema on each write leading to too many schema change requests. Even if these issues are avoided, multiple outputs will be less performant than a single merged output as it can send too many requests.
> >

Reviewer: ```suggestion
> We do not recommend using multiple outputs to write to the same dataset because it can cause several issues. Each output tries to create the Power BI dataset independently which can result in multiple datasets with the same name. Additionally, if the outputs don't have consistent schemas, the dataset changes the schema on each write, which leads to too many schema change requests. Even if these issues are avoided, multiple outputs will be less performant than a single merged output.
```

Power BI uses the first-in, first-out (FIFO) retention policy. Data will collect in a table until it hits 200,000 rows.

> [!NOTE]

> > We do not recommend using multiple outputs to write to the same dataset because it can cause several issues. Each output tries to create the Power BI dataset independently which can result in multiple datasets with the same name. Additionally, if the outputs don't have consistent schemas, the dataset changes the schema on each write, which leads to too many schema change requests. Even if these issues are avoided, multiple outputs will be less performant than a single merged output.

Reviewer 2: Using multiple outputs to ....... is not recommended.
